src/common/HTTPRequest.ts

```diff
@@ -216,7 +216,7 @@ export class HTTPRequest {
    */
   private interceptResolution(): [InterceptResolutionStrategy, number?] {
     if (!this._allowInterception) return ['disabled'];
-    if (this._interceptionHandled) return ['alreay-handled'];
+    if (this._interceptionHandled) return ['already-handled'];
```

Comment:
The `InterceptResolutionStrategy` must be updated too.

Reply:
ohh , I will check it once